AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -185,24 +185,55 @@ - (void)removePlayerItemObservers:(AVPlayerItem *)playerItem
   [notificationCenter removeObserver:self name:AVPlayerItemNewErrorLogEntryNotification object:playerItem];
 }
 
+- (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
+{
+    // All subnodes should taking the whole node frame
+    CGSize maxSize = constrainedSize.max;
+    if (!CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero)) {
+        maxSize = self.preferredFrameSize;
+    }
+    
+    // Prevent crashes through if infinite width or height
+    if (isinf(maxSize.width) || isinf(maxSize.height)) {
```

Comment:
What about the case when the minimum size is nonzero and the max size is inf?  should we use the min size?